magento/magento2
# Fix-issue: 26022 - Admin attributes Advanced Attribute Properties icon doesn't change while collapsing block

2.3-update





### Description (*)

Admin attributes Advanced Attribute Properties icon doesn't change while collapsing block fixed

### Fixed Issues (if relevant)

1. magento/magento2#26022: Admin attributes Advanced Attribute Properties icon doesn't change while collapsing block

### Manual testing scenarios (*)

1. Login to admin
2. Store -> Attributes -> products -> properties -> Advanced attribute properties
3. check the icon of Advanced attribute properties block

### Questions or comments


### Contribution checklist (*)
